xls/modules/zstd/frame_header_test.cc

```diff
@@ -58,9 +58,19 @@ enum FrameHeaderStatus : uint8_t {
 };
 
 struct ZstdFrameHeader {
-  absl::Span<const uint8_t> kBuffer;
   ZSTD_frameHeader kHeader;
   size_t kResult;
+
+  absl::Span<const uint8_t> kBuffer() const {
```

Comment:
google style is just buffer() since the member is buffer_

Reply:
Changed from `kBuffer()` to `buffer()`